src/passes/StringLifting.cpp

```diff
@@ -66,7 +66,14 @@ struct StringLifting : public Pass {
         continue;
       }
       if (global->module == stringConstsModule) {
-        importedStrings[global->name] = global->base;
+        // Encode from WTF-8 to WTF-16.
+        auto wtf8 = global->base;
+        std::stringstream wtf16;
+        bool valid = String::convertWTF8ToWTF16(wtf16, wtf8.str);
+        if (!valid) {
+          Fatal() << "Bad string to lift: " << wtf8;
+        }
+        importedStrings[global->name] = wtf16.str();
```

Comment:
This moved from below. Strings from consts, and strings from the section, now get different handling.